pkg/source/source.go

```diff
@@ -219,7 +204,7 @@ type Channel struct {
 	// Source is the source channel to fetch GenericEvents
 	Source <-chan event.GenericEvent
 
-	// stop is to end ongoing goroutine, and close the channels
+	// Stop is to end ongoing goroutine, and close the channels
```

Comment:
```suggestion
	// stop is to end ongoing goroutine, and close the channels
```
nit: given the field name is lower case?